src/library/Uno.Cupertino/Styles/Application/ColorPalette.xaml

```diff
@@ -52,6 +52,7 @@
 			<Color x:Key="SeparatorColor">#493c3c43</Color>
 			<Color x:Key="OpaqueSeparatorColor">#c6c6c8</Color>
 			<Color x:Key="LinkColor">#007aff</Color>
+			<Color x:Key="RadioButtonBorderColor">#007BFF</Color>
```

Comment:
Same here and for the dark theme. We should define ActiveRadioButtonBorderBrush and InactiveRadioButtonBorderBrush inside RadioButton.xaml since this is not something cupertino defined by itself